src/definitions.jl

```diff
@@ -278,7 +278,7 @@ plan_ifft(x::AbstractArray, region; kws...) =
 plan_ifft!(x::AbstractArray, region; kws...) =
     ScaledPlan(plan_bfft!(x, region; kws...), normalization(x, region))
 
-plan_inv(p::ScaledPlan) = ScaledPlan(plan_inv(p.p), inv(p.scale))
+plan_inv(p::ScaledPlan) = ScaledPlan(inv(p.p), inv(p.scale))
```

Comment:
So to understand this correctly: that was a bug in the package and when fixing it tests would fail currently (without the changes in test/testplans.jl)?

Reply:
That's right: fixing this without making changes to the test plans caused things to fail. 

I'm not sure to what extent I'd call the original version a bug in the package, but it did lead to ignoring the `pinv` field of any plan inside a scaled plan: so for example given a plan `P` a new inverse plan would be made for `2 * P`, `3 * P`, `4 * P`, etc., which is why the caches made by the test plans were just ignored.

Reply:
OK. I just wanted to make sure that this change is covered by the tests, ie that reverting it would cause errors.